docs/DesignDocs/SGX_Endorsements_V2.md

```diff
@@ -0,0 +1,222 @@
+SGX Attestation Endorsements V2 Updates
+================
+
+Intel SGX released v2 of the Provision Certification Service for ECDSA Attestation.  The
+following APIs were affected:
+
+1. Get TCB Info.
+2. Get Quoting Enclave Identity.
+
+[API Version 1](https://api.portal.trustedservices.intel.com/documentation#pcs-certificate)
+
+[API Version 2](https://api.portal.trustedservices.intel.com/documentation#pcs-certificate-v2)
+
+This document describes the changes for supporting version 2 of the API as well as keeping
+the existing support for version 1 of the API.  This document assumes the user is familiar with
+the SGX attestation endorsements and how it is used during SGX quote verification.
+
+Motivation
+----------
+
+This change is required for customers who would like to use the v2 API web endpoints. Note that the Azure PCK
+Caching Service and the Azure DCAP Client will also need to be updated and it is outside of the this specification.
+
+User Experience
+---------------
+
+The user experience does not change, it should be seamless to the user.  The caller will be able to identify the version of the data by checking the existing `version` field.
+
+The quote verifier will specify whether to use the v1 or v2 Azure PCK Caching Service web API endpoints.  The
+quote verification logic will be able to determine the version of the API by checking the `version` field in endorsement data.
```

Comment:
How does a quote verifier provide this option?

Reply:
This depends on how the version of the DCAP provider installed.

Reply:
Had a side discussion on this, will clarify the quote verifier as an entity that does the quote verification.